src/main/java/spoon/NoClasspathWorkaround.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * SPDX-License-Identifier: (MIT OR CECILL-C)
+ *
+ * Copyright (C) 2006-2019 INRIA and contributors
+ *
+ * Spoon is available either under the terms of the MIT License (see LICENSE-MIT.txt) of the Cecill-C License (see LICENSE-CECILL-C.txt). You as the user are entitled to choose the terms under which to adopt Spoon.
+ */
+package spoon;
```

Comment:
I hate to come with yet another thing after not saying anything for so long, but should we not perhaps put this in the internal package? It seems like something we don't want to share with the world, but only use internally in Spoon.

Reply:
Actually, `spoon.testing.utils` is probably the place we want to put it, considering the naming.

Reply:
Using a testing package seems wrong for this. As this is not really related to testing or? An internal package, yes indeed.

Reply:
> Using a testing package seems wrong for this. As this is not really related to testing or?

I have no idea of what last week me was thinking. It was close to midnight so I suspect: not much.

> An internal package, yes indeed.

Yes. That.

Reply:
How about we do this in `spoon.experimental` and see if we have enough classes to create an internal package in the future?

Reply:
@MartinWitt Ugh, missed this comment.

But yes, sure. Essentially, if you put it anywhere where we can freely break it I'm satisfied.

Reply:
> Ugh, missed this comment.


No worries, I also had a lengthy summer vacation and enjoyed the nice weather.
I moved the class to the experimental package and added `Experimental` and `Internal`. More evidence that we can and will break this annotation's package is not possible.